deploy/05_erc721.ts

```diff
@@ -91,9 +91,35 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
     console.log('"CxipERC721" is already deployed.');
   }
 
+  // HolographLegacyERC721
+  const futureHolographLegacyErc721Address = await genesisDeriveFutureAddress(
+    hre,
+    salt,
+    'HolographLegacyERC721',
+    generateInitCode(['address'], [deployerAddress])
+  );
+  console.log('the future "CxipERC721" address is', futureHolographLegacyErc721Address);
```

Comment:
@alexanderattar this says cxipERC721 when it should be HolographLegacyERC721

Reply:
weird are you sure you pulled the latest.. this one is already updated on my end
![image](https://github.com/holographxyz/holograph-protocol/assets/1504794/5ee539cf-8218-445c-9e5c-ae2c1cc87f7b)